app/models/manageiq/providers/openshift/container_manager/refresh_parser.rb

```diff
@@ -7,10 +7,21 @@ def ems_inv_to_hashes(inventory)
         get_routes(inventory)
         get_builds(inventory)
         get_build_pods(inventory)
+        get_openshift_images(inventory)
         EmsRefresh.log_inv_debug_trace(@data, "data:")
         @data
       end
 
+      def get_openshift_images(inventory)
+        process_collection(inventory["image"], :container_images) { |n| parse_openshift_image(n) }
+        @data[:container_images].uniq! { |n| n[:image_ref] }
```

Comment:
Yes because `get_images` from the kubernetes refresh parser will push its images to `@data` and then `process_collection` here will also do that so each image that will be identified by both will be inserted twice (but it will be the same object). So we need to filter them out. In `@data_index` we won't have any duplications because we are just updating images that were added before. Another solution could be to do `@data[:container_images] = []` before the `process_collection` here and I think I will change to it now.